apps/src/code-studio/pd/workshop_dashboard/components/workshop_form.jsx

```diff
@@ -619,6 +693,49 @@ const WorkshopForm = React.createClass({
           <Row>
             <Col sm={10}>
               <FormGroup>
+                <ControlLabel>
+                  Workshop Type Options
+                  &nbsp;<a onClick={this.toggleTypeOptionsHelpDisplay}>(help)</a>
```

Comment:
Do you need the `&nbsp;` here? Won't the newline inject a (breakable) space anyway?